freqtrade/utils/get_sides.py

```diff
@@ -0,0 +1,5 @@
+from typing import Tuple
+
+
+def get_sides(is_short: bool) -> Tuple[str, str]:
+    return ("sell", "buy") if is_short else ("buy", "sell")
```

Comment:
I'm sceptical of generic "utils" packages/modules/... (or anything named similarily) ... as it'll end up as the "trashcan" for everything that can't go anywhere (i've been looking for that too very often, but usually finding a "properly named" place results in a better solution - or it'll end up in misc.py).

is your intend to use this also outside of tests in the future, or is it limited to tests?
If it's limited to tests (and has no benefit to the remainder of freqtrade) - then maybe moving it into the tests directory would make sense...

Reply:
I moved it to conftest